### PR TITLE
Issue #436 Edited uninstall instructions

### LINKED ISSUE #436: Edit Minishift uninstall instructions
In the README there is a new section with uninstall instructions, needed editing so did some editing. PR coming in 3... 2... 1...

### PR DESCRIPTION
**Pull request by thatdocslady**
Fixed #436 

This PR addresses proofreading and usability of the uninstall section of the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,24 +101,24 @@ You can now install the latest beta version of minishift.
...
+1. Delete the Minishift VM and any VM-specific files:
 
         $ minishift delete
 
-   This will remove everything under `MINISHIFT_HOME/.minishift/machines/minishift`.
-   Other cached data and the [persistent configuration](./docs/using.md#persistent-configuration) will
-   be kept.
-1. If you want to remove Minishift completely, you also need to remove everything under `MINISHIFT_HOME`
+   This command deletes everything in the `MINISHIFT_HOME/.minishift/machines/minishift` directory.
+   Other cached data and the [persistent configuration](./docs/using.md#persistent-configuration) is not removed.
```
Comment (budhrg) [inline README.md]: `are not removed`.

**Comment by budhrg** [comment]
LGTM :+1:

File: README.md (hunk 1 above)
Comment (thatdocslady) [inline README.md]: whoops! thanks :)

**Comment by thatdocslady** [comment]
@hferentschik I thought you added me to whitelist so that this rubbish with the CI bot won't come up again?

**Comment by hferentschik** [comment]
Merged, thanks.

**Comment by hferentschik** [comment]
@budhrg - lets add @thatdocslady  and @Preeticp  to here  - https://github.com/minishift/minishift-ci-jobs/blob/master/minishift-ci-index.yaml#L20 - as well

**Pull request closed without merging**